src/utils.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { toString } from './utils';
+
+describe('#toString', () => {
+  it('test custom object toString function', () => {
+    const value = {
+      toString() {
+        return 'foo';
+      },
+    };
+    expect(toString(value)).toEqual('foo');
+  });
+
+  it('test custom object toString not "function"', () => {
```

Comment:
Not sure if this one is good, what do you think?

```suggestion
  it('calls Object.prototype.toString() when custom one is not function', () => {
```